content/source/docs/enterprise/run/index.html.md

```diff
@@ -82,19 +91,40 @@ Most importantly, it shows:
 - A timeline of events related to the run.
 - The output from both the `terraform plan` and `terraform apply` commands, if applicable. You can hide or reveal these as needed; they default to visible if the command is currently running, and hidden if the command has finished.
 
-If the run is still in progress and you have write access to the workspace, there are controls for interacting with the run at the bottom of the page. Depending on the state of the run, the following buttons might be available:
+## Interacting with Runs
+
+-> **API:** See [the Runs API](../api/run.html).
+
+In workspaces where you have write permissions, run pages include controls for interacting with the run at the bottom of the page. Depending on the state of the run, the following buttons might be available:
+
+Button              | Available when:
+--------------------|----------------
+Add Comment         | Always.
+Confirm & Apply     | A plan needs confirmation.
+Override & Continue | A soft-mandatory policy failed (only available for owners team).
+Discard Run         | A plan needs confirmation or a soft-mandatory policy failed.
+Cancel Run          | A plan or apply is currently running.
+Force Cancel Run    | A plan or apply was canceled, but something went wrong and TFE couldn't end the run gracefully (only available with workspace admin permissions).
```

Comment:
This seems fine if we're in the habit of specifying the lowest level required - otherwise, perhaps "workspace admin or organization owner")

Reply:
At the moment, every workspace shows the owners team as having admin permissions, so I think that can stay.